types/index.d.ts

```diff
@@ -2428,6 +2428,14 @@ export interface MathJsInstance extends MathJsFactory {
    * Probability functions
    ************************************************************************/
 
+  /**
+   * Compute the nth Bernoulli number
+   * @param n  index
+   * @returns  nth Bernoulli number
+   */
+  bernoulli<T extends number | Fraction | BigNumber>(n: T): NoLiteralType<T>
```

Comment:
Can you also add a type definition for the chained version of this function?

Reply:
oops yeah so many pieces to adding a method... done now.